contracts/eden/src/actions/genesis.cpp

```diff
@@ -60,6 +61,9 @@ namespace eden
 
          induction_id++;
       }
+
+      atomicassets::init_collection(atomic_assets_account, get_self(), collection_name,
+                                    schema_name);
```

Comment:
can we change the schema_name to `members` in our constants as well?

Reply:
nevermind, I just remembered that the schema name is set on compile time on `EDEN_SCHEMA_NAME`